lib/ui/screen/notifications_screen.dart

```diff
@@ -1,56 +1,218 @@
+                            'group_kick'
+                            ? Text(
+                            '${notification.user
+                                .name} kicked you out of ${notification
+                                .entityObj != null
+                                ? notification.entityObj.name
+                                : ""}' )
+                            : notification
+                            .filter ==
+                            'group_queue_add'
+                            ? Text(
+                            '${notification.user.name} added you to the queue' )
+                            : notification
+                            .filter ==
+                            'group_queue_approve'
+                            ? Text(
+                            '${notification.user
+                                .name} approved to add you to the queue' )
+                            : notification.filter ==
+                            'group_queue_reject'
+                            ? Text(
+                            '${notification.user
+                                .name} rejected your request to add you to the queue' )
+                            : notification.filter ==
+                            'create_admin_group'
+                            ? Text(
+                            '${notification.user.name} created the group' )
+                            : notification.filter == 'group_join_reject'
+                            ? Text( '${notification.user
+                            .name} rejected your request to join the group' )
+                            : notification.filter == 'wire_happened' ? Text(
+                            'you have succesfully contributed ${notification
+                                .params.amount} to @${notification.params
+                                .username}' ) : notification.filter == 'remind'
+                            ? Text(
+                            '${notification.user.name} reposted ${notification
+                                .params.title}' )
+                            : notification.filter == 'group_join_accept' ? Text(
+                            '${notification.user.name} has joined ${notification
+                                .entityObj != null
+                                ? notification.entityObj.name
+                                : ""}' ) : notification.filter == 'boost_accept'
+                            ? Text( '${notification.user
+                            .name} have accepted the boost' )
+                            : notification.filter == 'boost_reject'
+                            ? Text( '${notification.user
+                            .name} have rejected the boost' )
+                            : Container( ),
+                      ],
+                    ),
+                  ),
+                ),
+              ),
+            ),
+          ],
+        ),
```

Comment:
Why build item have 100 lines of code .. why it’s not refactored widget by widget ?